.github/workflows/build.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
           - {os: ubuntu-24.04-arm, variant: debian, name: Debian ARM, artifact: debian-arm}
           - {os: ubuntu-24.04, variant: make-qmi, name: Linux with QMI, artifact: linux-qmi-x64}
           - {os: ubuntu-24.04-arm, variant: make-qmi, name: Linux ARM with QMI, artifact: linux-qmi-arm}
+          - {os: ubuntu-24.04, variant: make-openwrt, name: OpenWrt, artifact: openwrt-x64}
+          - {os: ubuntu-24.04-arm, variant: make-openwrt, name: OpenWrt ARM64, artifact: openwrt-arm64}
```

Comment:
If you want to build for OpenWRT, please build on OpenWRT or use the OpenWRT SDK to build, and the executable will be target-specific (it means you must use a full target like `mediatek-filogic_gcc-13.3.0_musl` instead of a simple `openwrt-arm64`).